client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java

```diff
@@ -174,30 +174,19 @@ private ActionListener<MLModelGetResponse> getMlGetModelResponseActionListener(A
         ActionListener<MLModelGetResponse> internalListener = ActionListener.wrap(predictionResponse -> {
             listener.onResponse(predictionResponse.getMlModel());
         }, listener::onFailure);
-        ActionListener<MLModelGetResponse> actionListener = wrapActionListener(internalListener, res -> {
-            MLModelGetResponse getResponse = MLModelGetResponse.fromActionResponse(res);
-            return getResponse;
-        });
-        return actionListener;
+        return wrapActionListener(internalListener, MLModelGetResponse::fromActionResponse);
     }
 
     @Override
     public void deleteModel(String modelId, ActionListener<DeleteResponse> listener) {
         MLModelDeleteRequest mlModelDeleteRequest = MLModelDeleteRequest.builder().modelId(modelId).build();
 
-        client.execute(MLModelDeleteAction.INSTANCE, mlModelDeleteRequest, ActionListener.wrap(deleteResponse -> {
-            listener.onResponse(deleteResponse);
-        }, listener::onFailure));
+        client.execute(MLModelDeleteAction.INSTANCE, mlModelDeleteRequest, ActionListener.wrap(listener::onResponse, listener::onFailure));
```

Comment:
I'm missing the need to wrap the action listener here without doing anything else with it.

(Same comment for multiple of these.)

Reply:
Sorry I didn't clearly follow you here.

Reply:
^^^ what @mingshl said.  No need to wrap it, it adds no value.

Reply:
Well I haven't changed much here. Just added minor changes of code refactoring what intellij editor suggested :D. So I'll keep that way for now. If we need to change, I'll raise another PR only targeting to remove the wrapper.